index.html

```diff
@@ -332,23 +437,193 @@ <h4>SvgRenderingTemplate</h4>
       "name": "Bachelor of Science and Arts"
     }
   },
-  <span class="highlight">"renderMethod": [{
-    "id": "https://example.edu/credentials/BachelorDegree.svg",
-    "type": "SvgRenderingTemplate",
-    "name": "Portrait Mode",
-    "css3MediaQuery": "@media (orientation: portrait)",
-    "digestMultibase": "zQmAPdhyxzznFCwYxAp2dRerWC85Wg6wFl9G270iEu5h6JqW"
-  }]
+  <span class="highlight">"renderMethod": {
+    "type": "TemplateRenderMethod",
+    "renderSuite": "svg-mustache",
+    "template": {
+      "id": "https://example.edu/credentials/BachelorDegree.svg",
+      "mediaType": "image/svg",
+      "digestMultibase": "zQmerWC85Wg6wFl9znFCwYxApG270iEu5h6JqWAPdhyxz2dR",
+      "renderProperty": [
+        "/issuer", "/validFrom", "/credentialSubject/degree/name"
+      ]
+    }
+  }
   </span>
 }
         </pre>
 
         <p>
-In the example above, the <a>issuer</a> has provided an SVG rendering template
-for a bachelor's degree that will be filled in with the information in the
-<a>verifiable credential</a>. The rendering is preferred when the device or
-paper is in a portrait (display is taller than it is wider) orientation.
+In the example above, the [=issuer=] has provided a Mustache-based SVG rendering
+template for a Bachelor's degree that will be filled in with specific
+information from the [=verifiable credential=] listed in `renderProperty`.
         </p>
+
+        <section>
+          <h3>The `svg-mustache` Render Suite</h3>
+          <p>
+The `svg-mustache` render suite uses the Mustache templating language to
+modify an SVG file, which is then used to render a visual representation
+of the [=verifiable credential=].
+          </p>
+
+          <p>
+In the example below, a fully embedded SVG file is used as the rendering
+template.
+          </p>
+
+          <pre class="example nohighlight"
+               title="Basic usage of the svg-mustache render suite">
+{
+  <span class="comment">...</span>
+  "renderMethod": {
+    "type": "TemplateRenderMethod",
+    "renderSuite": "svg-mustache",
+    <span class="comment">// the SVG file is embedded in the VC</span>
+    "template": "data:image/svg;base64,Qjei89...3jZpW"
```

Comment:
```suggestion
    "template": "data:image/svg+xml;base64,Qjei89...3jZpW"
```

Reply:
IIRC, @davidlehn had discovered that base64'ing SVGs often bloats the actual size. It also makes processing slower (due to required decoding).

However, there are other escaping requirements to deal with double quotes and whitespace.

Reply:
My understanding is that we're constrained by the data URL spec here, which only understands either a base64-encoding of or a direct expression of the content according to the media type. So unless there's another media type (or parameter) we could leverage that would indicate that the direct content is "escaped" --  it seems that base64-encoding is a requirement here.